solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBSolrClient.java

```diff
@@ -60,10 +61,18 @@
 import org.apache.solr.common.util.ObjectReleaseTracker;
 import org.apache.solr.common.util.SolrNamedThreadFactory;
 import org.apache.solr.common.util.URLUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.slf4j.MDC;
 
 public abstract class LBSolrClient extends SolrClient {
 
+  private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+  protected static final String UPDATE_LIVE_SERVER_MESSAGE = "Updated alive server list";
+
+  private static final String UPDATE_LIVE_SERVER_LOG = UPDATE_LIVE_SERVER_MESSAGE + ": {}";
```

Comment:
is this constant really necessary?

Reply:
The first one is to prevent the test from breaking should someone alter the log message.  The second is to make precommit happy (no string concatenation is allowed in a log message it seems).

Reply:
see validate-log-calls.gradle.  You could have added "//nowarn".
Personally I hate over-use of constants; it adds needless indirection to reading code.